npm-api-maker/src/can-can-loader.jsx

```diff
@@ -0,0 +1,32 @@
+import ApiMakerEventEmitterListener from "./event-emitter-listener"
+
+export default class ApiMakerCanCanLoader extends React.Component {
+  static propTypes = PropTypesExact({
+    abilities: PropTypes.array.isRequired,
+    component: PropTypes.object.isRequired
+  })
+
+  componentDidMount() {
+    this.loadAbilities()
+  }
+
+  async loadAbilities() {
+    const canCan = CanCan.current()
+
+    await canCan.loadAbilities(this.props.abilities)
+
+    this.props.component.shape.set({canCan})
+  }
+
+  render() {
+    return (
+      <ApiMakerEventEmitterListener events={CanCan.current().events} event="onResetAbilities" onCalled={() => this.onResetAbilities()} />
+    )
+  }
```

Comment:
Just in case another developer decides to rename this variable or something in the future.
```suggestion
  render() {
    const canCan = CanCan.current()
    const events = digg(canCan, "events")

    return (
      <ApiMakerEventEmitterListener events={events} event="onResetAbilities" onCalled={() => this.onResetAbilities()} />
    )
  }
```